docs/turborepo-migration-guide.md

```
@@ -0,0 +1,1046 @@
+```json
+{
+  "name": "@churchthreads/ui",
+  "version": "0.0.1",
+  "private": true,
+  "exports": {
+    ".": "./index.ts"
+  }
+}
+```
+
+2. **Create `index.ts`** with your exports.
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)